docker/run.sh.example

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+docker run --rm -d \
+	--name aif-validator \
+    	-it aif-validator
```

Comment:
Maybe this is just part of the TBDs in this, or it's because I've never used Docker, but does this actually run the validator?  I would think that it would need to invoke the `target/appassembler/bin/validateAIF` script, which invokes the `ValidateAIF` class file.

Reply:
Yep, this is part of the TBD. We first need to decide how this docker container is going to be used. Because we have not decided this yet I made the docker container not do anything except tail continuously so  the reviewer can go into the container to review the installation of the validator. I mention this in the README at the very end.